src/browser/api_protocol/api_handlers/middleware_entity_existence.ts

```diff
@@ -23,6 +23,7 @@ import { windowApiMap } from './window.js';
 const apisToIgnore = new Set([
     // Application
     'create-application',
+    'join-window-group',
```

Comment:
:neckbeard: :neckbeard: :neckbeard: ⚠️ This **brakes** alphabetical order 😱

Reply:
I brake for breaking the order!  🛑